src/utils/tokens.tsx

```diff
@@ -121,6 +121,33 @@ export async function getOwnedTokenAccounts(
     });
 }
 
+export async function getMultipleTokenAccountInfo(
+  connection: Connection,
+  ownerAddress: PublicKey,
+) {
+  let [splAccounts, account] = await Promise.all([
+    getOwnedTokenAccounts(connection, ownerAddress),
+    connection.getAccountInfo(ownerAddress),
+  ]);
+  let accounts = splAccounts.map(value => {
+    return value.publicKey;
+  });
+  let response = await getMultipleSolanaAccounts(connection, accounts);
```

Comment:
This appears to just be the same function as `getTokenAccountInfo` but with an *added* and unnecessary call to `getMultipleSolanaAccounts`?

Reply:
This uses the response from `getMultipleSolanaAccounts` instead of just a single account to return a full list. Is that still unnecessary?

Reply:
Reverted this